src/reducers/subCategorizeAll.ts

```diff
@@ -42,6 +42,9 @@ const subCategorizeAll = (state: State) => {
   const children = getChildrenRanked(state, pathToContext(simplifyPath(state, path)))
   const pathParent = cursor.length > 1 ? cursorParent : HOME_PATH
 
+  // filter out meta children that should not be moved
+  const filteredChildren = children.filter(child => !SUBCATEGORIZE_ALL_META_EXCEPTIONS.includes(child.value))
```

Comment:
Once the exceptions are hardcoded in the tests, we can move them out of constants into this module.

Reply:
Sure will change this.